consul-lambda/consul-lambda-registrator/trigger.go

```diff
@@ -57,7 +67,7 @@ type RequestParameters struct {
 }
 
 // AWSEventToEvents converts an AWSEvent to a list of Events required to sync Lambda state with Consul.
-func (e Environment) AWSEventToEvents(ctx context.Context, event AWSEvent) ([]Event, error) {
+func (env Environment) AWSEventToEvents(ctx context.Context, event AWSEvent) ([]Event, error) {
```

Comment:
nit: changing `e` to `env` for the `Environment` struct clutters this PR a bit and makes the functional changes less obvious. I think in the future, it might be better to make a separate PR for a cosmetic change like this.